config/data/langs.toml

```diff
@@ -1042,6 +1042,23 @@ main :: proc() {
 }
 '''
 
+[PARI-GP]
+args       = [ '/usr/bin/pari-gp', '-' ]
+experiment = 219
+size       = '16.3 MiB'
+version    = '2.15.5'
+website    = 'https://pari.math.u-bordeaux.fr'
+example    = '''
+\\ Printing
+print("Hello, World!")
+
+\\ Looping
+for (i = 0, 9, print(i))
+
+\\ Accessing arguments
+for (i = 1, #(args=readstr("input.txt")), print(args[i]))
```

Comment:
While the PR remains on idle terms, I'll attempt to simplify and bind this. Regardless, this one works.